active_link/templatetags/active_link_tags.py

```diff
@@ -39,23 +39,24 @@ def active_link(
         # Can't work without the request object.
         return css_inactive_class
 
-    if request.resolver_match.kwargs != {}:
-        # Capture the url kwargs to reverse against
-        kwargs.update(request.resolver_match.kwargs)
+    resolver_kwargs = getattr(request, 'resolver_match', None)
+    resolver_kwargs = getattr(resolver_kwargs, 'kwargs', {}) if resolver_kwargs else {}
+    kwargs.update(resolver_kwargs)
```

Comment:
This is obfuscating the code more than it actually helps.
Please rewrite to make it easier to read.

Reply:
I've made it more readable